tutorials/3d/environment_and_post_processing.rst

```diff
@@ -266,141 +420,388 @@ This can be simulated with Screen Space Ambient Occlusion. As you can see in the
+well. If you don't do this, the LUT texture will not affect the viewport
+correctly when used. You can preview how the 3D texture was imported by
+double-clicking it, in the FileSystem dock, then going to the inspector to flip
+through the texture's layers.
+
+You can use this neutral 33×33×33 LUT template as a base (right-click and choose
+**Save as…**):
+
+.. image:: img/environment_adjustments_3d_lut_template.webp
+
+With the above LUT template, after changing its import mode to **Texture3D**,
+set its number of **Horizontal** slices to ``33`` in the Import dock then click
+**Reimport**. If you load this LUT into the **Color Correction** property, you
+won't see any visible difference for now since this texture is designed to be a
+neutral starting point.
+
+This LUT template can be modified in an image editor to provide a different
+mood to the image. A common workflow is to place the LUT image next to a
+screenshot of the project's 3D viewport, then use an image editor to modify both
+the LUT image and the screenshot at the same time. The LUT can then be saved and
+applied to the game engine to perform the same color correction in real-time.
+
+For example, modifying the LUT template in an image editor to give it a
+"sepia" look results in the image on the right:
+
+.. image:: img/environment_adjustments_3d_lut_comparison.webp
+
+.. note::
+
+    Adjustments and color correction are applied *after* tonemapping.
+    This means the tonemapping properties defined above still have an effect
+    when adjustments are enabled.
+
+Camera attribute options
+------------------------
+
+Depth of Field / Far Blur
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This effect simulates focal distance on cameras. It blurs objects behind
+a given range. It has an initial **Distance** with a **Transition** region
+(in world units):
+
+.. image:: img/environment_dof_far.webp
+
+The **Amount** parameter controls the amount of blur. For larger blurs, tweaking
+the **Quality** may be needed in order to avoid artifacts.
+
+Depth of Field / Near Blur
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This effect simulates focal distance on cameras. It blurs objects close
+to the camera (acts in the opposite direction as far blur).
+It has an initial **Distance** with a **Transition** region (in world units):
+
+.. image:: img/environment_dof_near.webp
+
+The **Amount** parameter controls the amount of blur. For larger blurs, tweaking
+the **Quality** may be needed in order to avoid artifacts.
+
+It is common to use both blurs together to focus the viewer's attention on a
+given object, or create a so-called
+`"tilt shift" effect <https://en.wikipedia.org/wiki/Miniature_faking>`__.
+
+.. image:: img/environment_mixed_blur.webp
+
+.. note::
+
+    When using CameraAttributesPhysical instead of CameraAttributesPractical,
+    depth of field is automatically computed from the camera attributes' focus
+    distance, focal length, and aperture.
+
+Exposure
+^^^^^^^^
+
```

Comment:
I know there's just one slider but there should be at least a sentence or two here.